generic/ltp.py.data/ltp.yaml

```diff
@@ -1,5 +1,7 @@
 setup:
     mem_leak: 0
+    url: 'https://github.com/linux-test-project/ltp/archive/master.zip'
+    skipfileurl: None 
```

Comment:
this line seems to have a whitespace at the end

Reply:
sure